src/library/Text/propTypes.js

```diff
@@ -3,6 +3,13 @@ import { bool, node, number, oneOf, oneOfType, string } from 'prop-types';
 import { ALIGN, APPEARANCE, FONT_WEIGHT } from './constants';
 
 export const textPropTypes = {
+  /* TargetX Custom Props */
+  bold: bool,
```

Comment:
I'd consider `bold` an alias/shorthand for `fontWeight: bold`, so if we're going to do this, let's also support the `fontWeight` prop as a passthrough